data/en/isempty.json

```diff
@@ -4,13 +4,21 @@
 	"syntax": "isEmpty(value)",
 	"returns": "boolean",
 	"related": [],
-	"description": "",
+	"description": "Returns whether or not a given value is "empty". ",
```

Comment:
Quotes inside quotes will not work. They have to be masked with a leading slash: `\"`